example/models/non_bedinput_model/dataloader.py

```diff
@@ -9,13 +9,11 @@
 import pybedtools
 from pybedtools import BedTool
 
-from genomelake.extractors import BaseExtractor, FastaExtractor, one_hot_encode_sequence, NUM_SEQ_CHARS
 from pysam import FastaFile
 from concise.utils.position import extract_landmarks, ALL_LANDMARKS
 
-
 from kipoi.data import Dataset
-
+from kipoi.helper import BaseExtractor, FastaExtractor, NUM_SEQ_CHARS, one_hot_encode_sequence
```

Comment:
Could copy in BaseExtractor from genomelake here and use FastaStringExtractor from kipoiseq